app/scripts/models/reliers/oauth.js

```diff
@@ -171,6 +174,14 @@ define([
           }
           throw err;
         });
+    },
+
+    isTrusted: function () {
+      return this.get('trusted');
+    },
+
+    needsPermissions: function (account) {
```

Comment:
I'm waffling whether this should be on the relier on on the account. I can see it both ways.

Ask the account if the relier needs permission to access its information.
vs
Ask the relier if it needs permission to access the account.

The first makes more sense to me.